niworkflows/reports/tests/test_core.py

```diff
@@ -0,0 +1,55 @@
+''' Testing module for niworkflows.reports.core '''
+
+import os
+from pathlib import Path
+from pkg_resources import resource_filename
+import tempfile
+
+import pytest
+
+from ..core import Report
+
+
+@pytest.fixture
+def test_report():
+    test_data_path = resource_filename(
+        'niworkflows',
+        os.path.join('data', 'tests', 'work', 'reportlets'))
+    out_dir = tempfile.mkdtemp()
+
+    return Report(Path(test_data_path), Path(out_dir), 'fakeiuud',
+                  subject_id='01', packagename='fmriprep')
+
+
+@pytest.mark.parametrize(
+    "orderings,expected_entities,expected_value_combos",
+    [
+        (['session', 'task', 'run'],
+         ['task', 'run'],
+         [
+            ('faketask', None),
```

Comment:
Now that you are at it, could you add svgs (I'd be fine with a placeholder) so that the layout finds:

1. A `faketask2` with `run` = `None`
2. A `faketaskwithruns` with `run` = {1, 2}

?

Along the same lines, within this PR or in a new one, we should add tests for more possible layouts (e.g., one with sessions comes to mind).

Reply:
good idea, I've added more combinations, and another test that is easier to dynamically create different BIDS datasets.
When you say placeholder you mean like a minimal `svg` file that does not really look like a report that would go into fmriprep?